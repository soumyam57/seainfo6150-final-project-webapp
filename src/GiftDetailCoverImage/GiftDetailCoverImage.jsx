import React from 'react';
import PropTypes from "prop-types";
import styles from "./GiftDetailCoverImage.module.css";

const GiftDetailCoverImage = props => {
    return (
        <img className={styles.image} src={props.url} alt=""/>
    )
}

GiftDetailCoverImage.propTypes = {
    url: PropTypes.string.isRequired,
};

export default GiftDetailCoverImage;
