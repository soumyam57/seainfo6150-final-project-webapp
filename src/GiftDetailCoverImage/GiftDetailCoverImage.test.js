import React from "react";
import { render } from "@testing-library/react";
import GiftDetailCoverImage from "./GiftDetailCoverImage.jsx";

const imageurl="https://images-na.ssl-images-amazon.com/images/I/51mfwq0NDEL._AC_UX522_.jpg";

it("renders the GiftDetailCoverImage component correctly", () => {
  const { container } = render(
    <GiftDetailCoverImage
        url={imageurl}
    />
    );
  expect(container).toMatchSnapshot();
});
