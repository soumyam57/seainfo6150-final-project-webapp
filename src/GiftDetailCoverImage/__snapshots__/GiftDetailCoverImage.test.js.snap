// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the GiftDetailCoverImage component correctly 1`] = `
<div>
  <img
    alt=""
    class="image"
    src="https://images-na.ssl-images-amazon.com/images/I/51mfwq0NDEL._AC_UX522_.jpg"
  />
</div>
`;
